content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -0,0 +1,105 @@
+      }
+    }
+  }
+}
+```
+
+This will enable the cache server mode for an Asset Process where it will write out the cached archive files to `T:/o3de/cache_folder` for all FBX files.
+
+The setting key `/AssetProcessor/Settings/Server/enableCacheServer=true` enables the **Asset Processor** to run in "server mode". In this mode, the **Asset Processor** will detect source asset changes, process the product assets, archive the product files (in a ZIP archive), and save out the archive files in the remote folder.
+
+The setting key `/AssetProcessor/Settings/Server/cacheServerAddress=<remote_shared_path>` points to a remote folder that the server and all clients can read and write from, respectively. The transfer folder should already be set up before launching the **Asset Processor**.
+
+The setting key `/AssetProcessor/Settings/Server/ACS FBX Glob={}` object specifies that the FBX source assets as a file type to cache. There can be a number of entries specified in the settings registry where the entry’s title needs to start with the letters ACS such as "ACS Our Textures" and "ACS Audio Files". The "glob" pattern can be used to capture files by extension or some basic matching pattern. It is important to flag the entry with "checkServer" to true to enable the entry for caching.
+
+
+## Run the Other Asset Processors in ACS in Client Mode
+
+To run the **Asset Processor** in ACS mode as a client the machine will access to the remote folder, enable the cache system in client mode, and specify the asset file types to pull from the remote server.
+
+```
+{
+  "Amazon": {
+    "AssetProcessor": {
+      "Settings": {
+        "Server": {
+          "enableCacheServer": false,
+          "cacheServerAddress": "T:/o3de/cache_server",
+          "ACS FBX Glob": {
+            "glob": "*.fbx",
+            "checkServer":  true
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The setting key `/AssetProcessor/Settings/Server/enableCacheServer=false` enables the **Asset Processor** to run in `client mode`. In this mode, the **Asset Processor** will read the source asset changes, check the remote folder for the product archives, and process the asset if it was not found. It is possible to have a hybrid of remote and local source assets since end users will add new assets locally before submitting them to a remote source asset repository.
+
+The cache server address and ACS specifier blocks should match the **Asset Cache Server**.
+
+## Configuring an ACS Block
+
+The asset caching system is configured using opt-in patterns. There are many types of files that process faster than copying an archived file from a remote folder. The most common way to cache the products of a source asset file is using a "glob" wild card pattern like "*.png" and "*.wav" scan patterns. Another way is to add a regular expression to match the source assets that would take a long time to process using "pattern" such as "\/assets\/rock_[\w]*\.asset" to cache all the rock files.
```

Comment:
The formatting isn't working right here as-is.  I can't see a proper preview when making this suggestion because of how suggestions work, but I think putting code tags around those blocks should fix it

Reply:
I will play around with it, but it looks alright in the Hugo server page. These really should be in quotes since they are string values, but they could be placed in mono spaced blocks. Thoughts?

Reply:
I don't think it's spelled out in the style guide, but in general we've been using code tags for values and file names/paths broadly including string values.

Reply:
Updated